### ISSUE TITLE
Add size_in_bytes to efs_file_system data source.

### ISSUE THREAD

**Issue by rquadling** [terraform-providers/terraform-provider-aws#13123]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I need to make a backup of an EFS File System. As a consequence, I need to know how big to make the volume, but I am currently unable to access the current EFS volume size without resorting to scripting such as:
```
aws efs describe-file-systems --profile $(env) \
  | jq -SMc '.FileSystems[0].SizeInBytes.Value'
```

Whilst this works, this is only getting the current value and then I have to manipulate it to get it to be in GB, add a percentage, round appropriately, store it as a variable, etc.

So, the proposal is to make the SizeInBytes.Value available in the efs_file_system data source.

### New or Affected Resource(s)

* aws_efs_file_system (data source)

### Potential Terraform Configuration

There are no configuration changes. The proposal only provides a new attribute to the efs_file_system data source, and as such, is readonly.

### References

* #13125

**Comment by anGie44** [comment]
The fix for this has been merged and will release with version 2.61.0 of the Terraform AWS Provider, expected in this week's release.

**Comment by rquadling** [comment]
Thank you.
